themes/custom/droopler_theme/scss/layout/_modifiers.scss

```diff
@@ -428,18 +430,50 @@ $two-columned-header-line-margin: 0.625rem !default;
   padding-top: 0;
 }
 
-.theme-primary > .paragraph {
-  background: $color-agamemnon;
-}
-
-.theme-gray > .paragraph {
-  background: $color-paris;
-}
-
-.theme-secondary > .paragraph {
-  background: $color-odysseus;
+/*
+  Themes styling.
+ */
+$d-p-themes: () !default;
+$d-p-themes: map-merge(
+    (
+      primary: (
+        background: $color-agamemnon,
+        text: $color-odysseus,
+        setFieldsColor: true,
```

Comment:
raczej w scss nie używamy camelCase. setFieldsColor -> set-fields-color

Reply:
ok, to zmienie